### PR TITLE
plugins:foreman-docker - Release 2.1.1 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**
For 1.11+  - please hold off merging until I verify 1.11 works well with this

### CODE REVIEW

File: plugins/ruby-foreman-docker/debian/gem.list [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_docker-2.0.1.gem"
-GEMS="$GEMS https://rubygems.org/downloads/docker-api-1.17.0.gem"
+GEMS="https://rubygems.org/downloads/foreman_docker-2.1.0.gem"
+GEMS="$GEMS https://rubygems.org/downloads/docker-api-1.18.0.gem"
```
Comment (domcleal) [inline plugins/ruby-foreman-docker/debian/gem.list]: Probably worth updating this to the latest, matching what you're doing with the RPMs.

File: plugins/ruby-foreman-docker/debian/gem.list [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 GEMS="$GEMS https://rubygems.org/downloads/wicked-1.1.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/archive-tar-minitar-0.5.2.gem"
```
Comment (domcleal) [inline plugins/ruby-foreman-docker/debian/gem.list]: ... and therefore removing this too.

**Comment by dLobatog** [comment]
https://github.com/theforeman/foreman-docker/pull/159 - when I merge it, I'll release 2.1.1 and we'll be able to move forward with ths PR.

**Comment by dLobatog** [comment]
Updated

**Comment by mmoll** [comment]
Cherry-picked as d1b68cc into develop and a665029 into 1.11, gracias @dLobatog!
Building now.

**Pull request closed without merging**